lightning/src/onion_message/messenger.rs

```diff
@@ -265,14 +265,31 @@ impl Responder {
 		}
 	}
 
-	/// Creates the appropriate [`ResponseInstruction`] for a given response.
+	/// Creates a [`ResponseInstruction::WithoutReplyPath`] variant of [`ResponseInstruction`]
+	/// for a given response.
```

Comment:
No need to repeat the enum name. Let's aim to be less verbose by conveying the most information in fewer words.

```rust
/// Creates a [`ResponseInstruction::WithoutReplyPath`] for sending the given `response`.
```

Reply:
Done! [8d5f9dc](https://github.com/lightningdevkit/rust-lightning/commit/8d5f9dce49b9b2b49a2e67d52a3021325ac3fcb3)
Thanks!